app/client/components/Clickr.js

```diff
@@ -0,0 +1,95 @@
+
+import React from 'react';
+
+export default class Clickr extends React.Component {
+
+    constructor(props) {
```

Comment:
you can skip the constructor (React docs relates to ES6 but we can use the more modern class approach which is offered by ES7)
```
export class Clickr ... {
  state = {
    count: 0,
  }

  ...

}
```